ddtrace/bootstrap/sitecustomize.py

```diff
@@ -84,12 +86,12 @@ def update_patched_modules():
     if asbool(get_env("runtime_metrics", "enabled")):
         RuntimeWorker.enable()
 
-    opts = {}
+    opts = {}  # type: Dict[str, Any]
```

Comment:
I wanted to make this more specific since we're only specifying integer/boolean/string arguments to configure, but mypy does not like this, as it thinks all possible arguments to `configure` (which include Samplers, ContextProviders, Writers) can be called here.

Reply:
ah... I mean, makes sense I guess

Reply:
I wonder if you make it restrictive here, then cast to a `dict[str, Any]` before `configure(**opts)` ?